types/big.js/index.global.d.ts

```diff
@@ -0,0 +1,348 @@
+export type BigSource = number | string | Big;
```

Comment:
does this change anything about the tests? Do the tests already cover global and module usage?